src/components/App/App.js

```diff
@@ -1,26 +1,40 @@
 import React, { Component } from 'react';
 import Header from '../Header/Header';
 import Movies from '../Movies/Movies';
-// import Film from '../Film/Film';
+import Film from '../Film/Film';
 import movieData from './movie-data';
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      currentMovie: {},
+      isHome: true
     }
   }
 
+  goHome = () => {
+    this.setState({ currentMovie: {}, isHome: true})
+  }
+
+  selectMovie = () => {
+    this.setState({ currentMovie: {}, isHome: false });
+  }
+
   render() {
     return (
       <>
-        <Header />
+        <Header goHome={this.goHome}/>
         <div className="App">
-          <Movies
-            movies={this.state.movies}
-          />
+          {!this.state.isHome
```

Comment:
I'm very excited about our understanding of conditional rendering! I think it'll be incredibly helpful with loading and error handling!